### PR TITLE
feat(pages): add hero and footer to home page

### PR DESCRIPTION
**Pull request by davidholyko**
#### Summary

- Work Type: Feature

#### What I did

- Added footer
- Added hero with text animation
- Ripped Header out of Routes and into individual pages

#### Notes / Research / References

- Colors from https://colorlab.dev/
- onAnimationEnd JSX attribute https://reactjs.org/docs/events.html#animation-events

#### Before/After Preview

- After:
https://deploy-preview-50--binge.netlify.app/#/

#### Checklist

- [x] Branch is in format `TYPE/scope/description`
- [x] PR title is in format `TYPE(scope): description`
- [x] Commits are in semantic format
- [x] Has Summary

### CODE REVIEW

File: src/components/SeriesSearch/SeriesSearchForm.js [hunk 1]
```diff
@@ -11,13 +11,17 @@ export const SeriesSearchForm = () => {
...
   const onHandleChange = _.debounce(async (text) => {
     setIsLoading(true);
-    const {
-      data: { results },
-    } = await getTitleMatches(text);
-    setIsLoading(false);
 
-    results && results.length && setSearchResults(results);
-  }, 1000);
+    if (text.length) {
```
Comment (davidholyko) [inline src/components/SeriesSearch/SeriesSearchForm.js:15]: minor unrelated fix: Fixes searching empty string after backspacing

File: src/scss/App.scss [hunk 2]
```diff
@@ -14,7 +14,7 @@
   height: 10px;
   bottom: 0;
   left: 0;
-  position: absolute;
+  position: fixed;
```
Comment (davidholyko) [inline src/scss/App.scss:17]: minor unrelated fix: Fixes loading bar not always at bottom of viewport

File: src/scss/Footer.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
+.footer {
+  background-color: #00233b;
```
Comment (davidholyko) [inline src/scss/Footer.scss:2]: we will want to consolidate colors into a new `Variables.scss` and give semantic names to them at a later point

**Review by simonchen92 — APPROVED** [review]
LGTM, but I have one question. When I click on the deployed Netlify link, it is showing me a CORS error. Has that always been the case?
